fluent.ebnf

```diff
@@ -43,8 +43,8 @@ expression           ::= quoted-pattern
                       ;
 select-expression    ::= expression __ ' ->' __ variants-list;
 member-expression    ::= identifier '[' keyword ']';
-call-expression      ::= builtin '(' __ arglist? __ ')';
-arglist              ::= argument (__ ',' __ arglist)?;
+call-expression      ::= builtin '(' __ NL? arglist? __ NL? ')';
+arglist              ::= argument (__ NL? ',' __ NL? arglist)?;
```

Comment:
Unless I misunderstand ebnf, this will not give you what you want.

In particular, you're saying that call-expression may have any number of white spaces, then optionally one or more NL, then any number of whitespaces, then once more optionally one or more NL and then an argument.

If I'm correct, this means that you can have at maximum two pairs of NL's(one or more) separated by '__' but not three. Seems arbitrary.

Would it make more sense to follow what parser does and do:

```
__ ::= [ \t]*;
___ ::= [ \t\n\r]*;
call-expression ::= '(' ___ arglist? ___ ')';
arglist ::= argument ( ___ arglist)?;
```